src/textbooks/textbook-card/TextbooksCard.test.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
[nit]: Please remove this unnecessary React import

Reply:
Fixed